wordpress-mobile/gutenberg-mobile
# Image block Caption.

Part of #140

This PR implements the [changes on the main gutenberg](https://github.com/WordPress/gutenberg/pull/10616) repo to show the image caption on the image block.

![caption](https://user-images.githubusercontent.com/9772967/45844603-52648c80-bcf9-11e8-882b-4b42f675e8bc.gif)

To test:
- Checkout and update Gutenberg.
- Run the project.
- Check that the caption is visible.
- Check that you can edit and remove the caption.
- Check that the changes are visible on the HTML view.
